src/components/common/Main.js

```diff
@@ -62,6 +62,7 @@ class Main extends Component {
     }).isRequired,
     children: PropTypes.node.isRequired,
     fullScreen: PropTypes.bool,
+    id: PropTypes.string.isRequired,
```

Comment:
We don't set the id everytime, we should remove the `.isRequired` and set some defaultProp (`undefined` or `''`)